### PR TITLE
Prevent loading angular multiple times with webpack

### LINKED ISSUE #123: Warning: Tried to load angular more than once.
I first thought this Warning was not related to angular-oauth2 package but when I got rid of it in my project, then the Warning was gone. 

Is there anyone having this Warning as well? I have angular-oauth2 defined as **devDependency** in package.json and then I'm importing it with `require('angular-oauth2)`. I'm using **Webpack** as well, and I followed some Issues recalling this issue like [this one](https://github.com/webpack/webpack/issues/2134) but didn't really find a solution.

Thanks in advance.

### LINKED ISSUE #126: 'npm install' is broken
Hi,

I cloned and tried to 'npm install' the project but it does not work.

```
npm verb cwd C:\Work\Others\github\angular-oauth2
npm verb Windows_NT 10.0.16299
npm verb argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\me\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "--verbose"
npm verb node v6.10.3
npm verb npm  v5.6.0
npm ERR! Error while executing:
npm ERR! C:\Program Files\Git\cmd\git.EXE ls-remote -h -t ssh://git@github.com/Nikku/node-browserify.git
npm ERR!
npm ERR! Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR!
npm ERR! exited with error code: 128
npm verb exit [ 1, true ]
```

### **update**
I'm not familiar with browserify so I'm gonna take two blind guesses here :
 - `github.com/Nikku/node-browserify` package doesnt exist and might got renamed to **karma-browserify**.
 - Or it seems like `node-browserify` was actually from repo `https://github.com/substack/node-browserify/` which got itself renamed to  `https://github.com/browserify/browserify` ( See [browserify changelog](https://github.com/browserify/browserify/blob/ccc02139481fb7f107009c2bd60a5e6f03610a63/changelog.markdown) for this change between version 14.4.0 and 14.5.0 )

I looked at the source on how to change this but I have honestly no clue how to fix this. Any hints ?

ps: Issue #121 mentions it but I wanted to create a proper issue entry for this

### PR DESCRIPTION
**Pull request by erhardos**
Dependencies were unlocked and moved to devDependencies, fixes #123 
Update browserify fixes #126 
Also fixes tests to work with newest Chrome version.

### CODE REVIEW

**Comment by ruipenso** [comment]
Thanks @erhardos. I'll take a look ASAP.

File: package.json [hunk 1]
```diff
@@ -21,13 +21,12 @@
...
-    "angular": "1.5.9",
-    "angular-cookies": "1.5.9",
-    "query-string": "^1.0.0"
-  },
+  "dependencies": {},
   "devDependencies": {
+    "angular": "^1.5.9",
+    "angular-cookies": "^1.5.9",
     "angular-mocks": "1.5.9",
+    "browserify": "^14.5.0",
```
Comment (pgom) [inline package.json]: Why did you need to add `browserify`?

File: package.json [hunk 2]
```diff
@@ -48,8 +47,10 @@
     "karma-sinon": "^1.0.4",
     "lodash": "^4.0.0",
     "mocha": "^2.4.5",
+    "query-string": "^1.0.0",
     "should": "^4.6.0",
     "sinon": "^1.17.3",
+    "watchify": "^3.9.0",
```
Comment (pgom) [inline package.json]: Same as `browserify` dependency

File: package.json (hunk 1 above)
Comment (pgom) [reply package.json]: Found out why you needed to add this dependency.
